MUXSDKStats/MUXSDKStats/MUXSDKStats.h

```diff
@@ -46,6 +46,17 @@ FOUNDATION_EXPORT
 - (_Null_unspecified instancetype)init NS_UNAVAILABLE;
 + (_Null_unspecified instancetype)new NS_UNAVAILABLE;
 
+/*!
+ @method      monitorAVPlayerViewController:withPlayerName:customerData:
+ @abstract    Starts to monitor a given AVPlayerViewController.
+ @param       player An AVPlayerViewController to monitor
+ @param       name A name for this instance of the player
+ @param       customerData A MUXSDKCustomerrData object with player, video, and view metadata
```

Comment:
is the idea that MUXSDKCustomerData is going to contain a MUXCustomerPlayerData MUXCustomerVideoData and MUXCustomerViewData?

Reply:
Exactly